src/app/weather-display/weather-display.component.ts

```diff
@@ -16,29 +16,34 @@ import { DateTime } from 'luxon';
   styleUrl: './weather-display.component.scss'
 })
 export class WeatherDisplayComponent {
+  dateNow = DateTime.local().toISO();
   locationData: any = [];
   dailyData: any = [];
   place_data = [
     { name: 'Okehampton', lat: 50.7390, lon: -4.0032 },
     { name: 'Torbay', lat: 50.4517, lon: -3.5579 },
     { name: 'Woodbury', lat: 50.6768, lon: -3.4005 }
   ];
-  dateNow = DateTime.local().toISO();
+  
 
   service = inject(AppService)
 
-  constructor() {
+  constructor() {}
+
+  idealConditions(data:any) {
+    const wind = data.windSpeed10m;
+    const precip = data.probOfPrecipitation;
+    const vis = data.visibility;
+    return this.service.conditionHighlight(wind, precip, vis);
   }
 
   ngOnInit() {
-
     this.locationData.push(
```

Comment:
Not sure if you have some line length setting but this looks here to read. I'd suggest it should probably be:

```
this.locationData.push(this.service.getData('Okehampton'));
```

if it had to go over multiple lines I'd move the last curly brace so it's clearer where the push() closes

etc etc